crates/uv-build-backend/src/lib.rs

```diff
@@ -484,7 +484,12 @@ fn wheel_subdir_from_globs(
     let license_files_globs: Vec<_> = globs
         .iter()
         .map(|license_files| {
-            trace!("Including license files at: `{license_files}`");
+            trace!(
+                "Including {} at `{}` with `{}`",
+                globs_field,
+                src.user_display(),
```

Comment:
Is this the thing that will emit an empty string for the CWD? Can we make it emit `./` instead for that case?

Reply:
I have an idea, but it affects more places so I'll follow up